### DISCUSSION TITLE
how to activate the look of the x86 version on x64 ?

### DISCUSSION THREAD

**Discussion by luapfr** [rustdesk/rustdesk#8482] [General]
I think the look of the x86 version is cleaner, especially the File Manager, which has thinner borders.

**Comment by xlionjuan** [comment]
Flutter is more feature rich

**Reply by luapfr** [reply]
> Flutter is more feature rich



> Flutter is more feature rich

should improve some things
I use the file manager a lot and the current form is horrible because I use glasses and my Windows has a large scala.

https://github.com/rustdesk/rustdesk/discussions/8484
